system_tests/noxfile.py

```diff
@@ -379,8 +380,9 @@ def mtls_http(session):
     )
 
 
-@nox.session(python=PYTHON_VERSIONS_SYNC)
+@nox.session(python=PYTHON_VERSIONS_ASYNC)
```

Comment:
What's the context for this change? Could we use both versions to avoid losing coverage for `PYTHON_VERSIONS_SYNC`?

Reply:
The previous version of pluggable auth required async libraries.  I think Chuan updated it to not require that.  I guess we'll find out when the tests run